### PR TITLE
Implemented: Blog component for CMS to display the blogs of the users

### PR DESCRIPTION
**Pull request by ymaheshwari1**
To display the blogs of the user with content like title, text, image.

### CODE REVIEW

File: pages/Blog.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <no-ssr>
```
Comment (adityasharma7) [inline pages/Blog.vue:2]: Hi @ymaheshwari1 
I think we are still using the component without SSR and blog should be rendered from SSR.

**Comment by AshishSuhane** [comment]
@ymaheshwari1  we need to register store in default.json under local forage array

File: pages/Blog.vue (hunk 1 above)
Comment (AshishSuhane) [reply pages/Blog.vue:2]: Yes, we can remove no-ssr component.

**Comment by dixitdeepak** [comment]
Closing this MR, as we are moving to storefront UI support and will handle this in Storefront UI support work.

**Pull request closed without merging**